opt_out_http_api/tests/test_api.py

```diff
@@ -136,11 +197,12 @@ def test_opt_out_nothing_to_delete(self):
 
     @inlineCallbacks
     def test_opt_out_count_two_opt_outs(self):
+        self.backend.count()
         resp = yield self.api_count("/optouts/count")
         self.assertEqual(resp.code, 200)
         data = yield resp.json()
         self.assertEqual(data, {
-            "opt_out_count": 2,
+            "opt_out_count": 0,
```

Comment:
Since the test is called `count_two_opt_outs`, changing this to a `0` seems like the wrong solution. You probably want to call `self.backend.put(...)` twice to add two opt outs.